libs/watt/src/lib/components/picker/shared/watt-range-input.service.ts

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 import { Injectable } from '@angular/core';
-import { combineLatest, Observable, tap } from 'rxjs';
+import { combineLatest, fromEvent, map, Observable, tap } from 'rxjs';
```

Comment:
Suggestion: Maybe it would be better with a watt-range-inputV2.service, so it dont mix the two versions.

Reply:
It seems to be leftovers from an earlier implementation of v2. It's not used in the v2, so now I've removed it.